src/App/App.tsx

```diff
@@ -5,24 +5,32 @@ import {
 } from 'react-router-dom';
 
 import List from '../UserManagement/List';
-import Header from '../Header';
-import TermList from '../Jank/TermManagement/TermList';
-import ScorepadList from '../Jank/Scorepads/ScorepadList';
+import Menu from '../Menu/Menu';
+import { GlobalStyles } from './ui';
+import JankSwitch from '../Jank/JankSwitch';
+import Home from '../Home/Home';
 
 function App() {
   return (
     <BrowserRouter>
-      <Header />
```

Comment:
Instead of having a one-fits-all header, I created separate header components for this (because we need the header to be different on every page anyway).